.vscode/settings.json

```diff
@@ -1,4 +1,12 @@
 {
-  // Fixes tsconfig not found error in nested .eslintrc.json files with relative paths
-  "eslint.workingDirectories": ["packages/intranet-header-workspace"]
+  "eslint.workingDirectories": [
+    "packages/intranet-header-workspace"
+  ],
+  "editor.formatOnSave": true,
+  "editor.defaultFormatter": "esbenp.prettier-vscode",
+  "[mdx]": {
+    "editor.defaultFormatter": null,
+    "editor.formatOnSave": false
```

Comment:
Setting defaultFormatter to null should be sufficient according to https://zellwk.com/blog/prettier-disable-languages but it's not. So I disabled as well formatOnSave for MDX according to https://github.com/prettier/prettier-vscode